src/cheat.rs

```diff
@@ -61,17 +82,15 @@ fn parse_variable_line(line: &str) -> (&str, &str, Option<SuggestionOpts>) {
     let caps = re.captures(line).unwrap();
     let variable = caps.get(1).unwrap().as_str().trim();
     let mut command_plus_opts = caps.get(2).unwrap().as_str().split("---");
-    let command = command_plus_opts.next().unwrap();
-    let opts = match command_plus_opts.next() {
-        Some(o) => Some(parse_opts(o)),
-        None => None,
-    };
-    (variable, command, opts)
+    let command: &str = command_plus_opts.next().unwrap();
+    let command_option_string: Option<&str> = command_plus_opts.next();
```

Comment:
Maybe use this value directly instead of using let?

Reply:
can do.